src/scheduled/weeklyStatistics.ts

```diff
@@ -1,166 +1,167 @@
+			if (owner?.guildMember?.joinedAt && owner.guildMember.joinedAt > lastInterval) {
+				console.log(owner.guildMember.joinedAt.toDateString())
+				console.log(lastInterval.toDateString())
+				newMembers.add(owner.user?.id)
+				postsByNewMembers++
+			}
 
-				for (let i = 0; i < threads.size; i++) {
-					const thread = threads.at(i)!;
-					if (thread.members.me) {
-						const msgs = await thread.messages.fetch();
-						msgs.forEach((message) => {
-							if (message.author.id == client.user!.id) count++;
-							return;
-						});
-					}
+			thread.appliedTags.forEach((tag) => {
+				if (!unsortedTags[tag]) {
+					unsortedTags[tag] = {}
 				}
 
-				redirectsEmbed.setDescription(
-					`I sent <#${process.env.SUPPORT_AI_CHANNEL}> redirects in ${count}/${threads.size} support threads`
-				);
+				thread.appliedTags.forEach((subTag) => {
+					if (!unsortedTags[tag][subTag]) {
+						unsortedTags[tag][subTag] = 0
+					}
 
-				embeds.push(redirectsEmbed);
-			} else {
-				console.warn('No SUPPORT_SQUAD_CHANNEL enviroment variable was set. Skipping support-ai redirects.');
-			}
+					unsortedTags[tag][subTag]++
+				})
+			})
 		}
 
-		// Tags
-		{
-			let unsortedTags: { [tag: string]: { [subTag: string]: number } } = {};
-
-			threads.forEach((thread) => {
-				thread.appliedTags.forEach((tag) => {
-					if (!unsortedTags[tag]) {
-						unsortedTags[tag] = {};
-					}
-
-					thread.appliedTags.forEach((subTag) => {
-						if (!unsortedTags[tag][subTag]) {
-							unsortedTags[tag][subTag] = 0;
-						}
-
-						unsortedTags[tag][subTag]++;
-					});
-				});
-			});
-
-			let description = '';
-			let embedCount = 0;
-
-			let tags = Object.fromEntries(
-				Object.entries(unsortedTags)
-					.sort((a, b) => {
-						return unsortedTags[a[0]][a[0]] - unsortedTags[b[0]][b[0]];
-					})
-					.reverse()
-			);
-
-			for (const tagId in tags) {
-				const tagName = await getTagName(guild, forum.availableTags, tagId);
-				let localDescription = `**${tagName}** (${tags[tagId][tagId]})\n`;
-
-				/** Sub tags sorted descending by count, excluding tags that show up just once. */
-				const subTags = Object.entries(tags[tagId])
-					.sort(([, countA], [, countB]) => countB - countA)
-					.filter(([subTagId, count]) => subTagId !== tagId && count > 1);
-
-				if (subTags.length) {
-					const subDescriptions = [];
-					for (const [id, count] of subTags) {
-						const subTagName = await getTagName(guild, forum.availableTags, id);
-						subDescriptions.push(`${subTagName} (${count})`);
-					}
-					localDescription += `+ ${subDescriptions.join(' / ')}\n`;
+		const openEmbed = getDefaultEmbed()
+		openEmbed.setTitle('New posts')
+		openEmbed.setDescription(`${_threads[0].size} open posts\n${_threads[1].size} closed posts\n${linkedToDocs} (${linkedToDocs/threads.length*100}%) include a link to docs in first response\nAverage response time of ${Math.round(cumulativeResponse/threads.length /1000/60)} minutes`)
```

Comment:
I saw this message in Discord:
![image](https://github.com/user-attachments/assets/94e29248-d345-4f3f-ad47-9a496c31a457)

But 9/77 ≈ 11.7%, not 10.5%
`_threads[0].size` is ALL threads (open and closed) = 77
`_threads[1].size` is only closed threads = 9
`threads` is both combined = 77 + 9 = 86 (duplicating closed threads)

9/86 ≈ 10.5% which is where the number comes from